resources/docker-build_p2p/ssh.rc

```diff
@@ -4,8 +4,11 @@
 ### the IP or DN of the p2p server
 p2p_server=86.119.39.89
 
+### the port of the http service in the p2p server
+http_port=800
```

Comment:
Okay, now it looks better :)
I was confused since one variable had a value and the other haven't. So: Approved :+1: